### PR TITLE
Styling: Add V1 json style

### PR DESCRIPTION
**Pull request by Jeremy-Prior**
- Tested new sprite link and adjusted the json style to work

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 40 sec

File: django_project/webmapping/styles/minisass_style_v1.json [hunk 1]
```diff
@@ -0,0 +1,434 @@
+{
+  "version": 8,
+  "name": "MyCivitas",
```
Comment (NyakudyaA) [inline django_project/webmapping/styles/minisass_style_v1.json]: It should be minisass

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 17 sec

**Comment by Jeremy-Prior** [comment]
@NyakudyaA Made the correction

**Review by NyakudyaA — APPROVED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 5 min 12 sec

**Pull request merged**